sys/net/gnrc/routing/rpl/gnrc_rpl_auto_init.c

```diff
@@ -28,35 +28,37 @@
 
 void auto_init_gnrc_rpl(void)
 {
-#if (GNRC_NETIF_NUMOF == 1)
-    gnrc_netif_t *netif = gnrc_netif_iter(NULL);
-    if (netif == NULL) {
-        /* XXX this is just a work-around ideally this would happen with
-         * an `up` event of the interface */
-        LOG_INFO("Unable to auto-initialize RPL. No interfaces found.\n");
+    if (gnrc_netif_highlander()) {
+        gnrc_netif_t *netif = gnrc_netif_iter(NULL);
+        if (netif == NULL) {
+            /* XXX this is just a work-around ideally this would happen with
+             * an `up` event of the interface */
+            LOG_INFO("Unable to auto-initialize RPL. No interfaces found.\n");
+            return;
+        }
+        DEBUG("auto_init_gnrc_rpl: initializing RPL on interface %" PRIkernel_pid "\n",
+              netif->pid);
+        gnrc_rpl_init(netif->pid);
         return;
     }
-    DEBUG("auto_init_gnrc_rpl: initializing RPL on interface %" PRIkernel_pid "\n",
-          netif->pid);
-    gnrc_rpl_init(netif->pid);
-    return;
-#elif defined(GNRC_RPL_DEFAULT_NETIF)
-    if (gnrc_netif_get_by_pid(GNRC_RPL_DEFAULT_NETIF) != NULL) {
-        DEBUG("auto_init_gnrc_rpl: initializing RPL on interface %" PRIkernel_pid "\n",
-              GNRC_RPL_DEFAULT_NETIF);
-        gnrc_rpl_init(GNRC_RPL_DEFAULT_NETIF);
+    else if (IS_ACTIVE(GNRC_RPL_DEFAULT_NETIF)) {
+        if (gnrc_netif_get_by_pid(IS_ACTIVE(GNRC_RPL_DEFAULT_NETIF)) != NULL) {
+            DEBUG("auto_init_gnrc_rpl: initializing RPL on interface %" PRIkernel_pid "\n",
+                  IS_ACTIVE(GNRC_RPL_DEFAULT_NETIF));
+            gnrc_rpl_init(IS_ACTIVE(GNRC_RPL_DEFAULT_NETIF));
+            return;
+        }
+        /* XXX this is just a work-around ideally this would happen with
+         * an `up` event of the GNRC_RPL_DEFAULT_NETIF */
+        DEBUG("auto_init_gnrc_rpl: could not initialize RPL on interface %" PRIkernel_pid" - "
+              "interface does not exist\n", IS_ACTIVE(GNRC_RPL_DEFAULT_NETIF));
         return;
     }
-    /* XXX this is just a work-around ideally this would happen with
-     * an `up` event of the GNRC_RPL_DEFAULT_NETIF */
-    DEBUG("auto_init_gnrc_rpl: could not initialize RPL on interface %" PRIkernel_pid" - "
-          "interface does not exist\n", GNRC_RPL_DEFAULT_NETIF);
-    return;
-#else
-    /* XXX this is just a work-around ideally this should be defined in some
-     * run-time interface configuration */
-    DEBUG("auto_init_gnrc_rpl: please specify an interface by setting GNRC_RPL_DEFAULT_NETIF\n");
-#endif
+    else {
+        /* XXX this is just a work-around ideally this should be defined in some
+         * run-time interface configuration */
+        DEBUG("auto_init_gnrc_rpl: please specify an interface by setting GNRC_RPL_DEFAULT_NETIF\n");
```

Comment:
Line is [too long](https://travis-ci.org/github/RIOT-OS/RIOT/builds/660563543#L228):
```suggestion
        DEBUG("auto_init_gnrc_rpl: please specify an interface by setting "
              "GNRC_RPL_DEFAULT_NETIF\n");
```